source/plugin/blocks/modifying.rst

```diff
@@ -17,29 +17,27 @@ Modifying Blocks
 Changing a Blocks Type
```

Comment:
I know you didn't change this, but it's otherwise going to irritate me - can you add an apostrophe in "Block's" please - the block owns the type! :)